[docs] Adjust the Material Icons page design and formatting (#36937)
mui/material-ui: docs/data/material/components/material-icons/material-icons.md

<p class="description">2,100+ ready-to-use React Material Icons from the official website.</p>

{{"component": "modules/components/ComponentLinkHeader.js"}}

> <br/>

Author: Had to add this `<br/>` element because, for some reason, adding a line here wasn't adding the space 🤔

[@mui/icons-material](https://www.npmjs.com/package/@mui/icons-material)
includes the 2,100+ official [Material Icons](https://fonts.google.com/icons?icon.set=Material+Icons) converted to [`SvgIcon`](/material-ui/api/svg-icon/) components.
It depends on `@mui/material`, which requires Emotion packages.
Use the following command to install it:

```sh
npm install @mui/icons-material @mui/material @emotion/styled @emotion/react
```

See the [Installation](/material-ui/getting-started/installation/) page for additional docs about how to make sure everything is set up correctly.

<hr/>

> Browse through all the icons and find the one you need!

Reviewer: ```suggestion
Browse through the icons below to find the one you need.
```

> The search below support synonyms－for example, try searching for "hamburger" or "logout".

Reviewer: ```suggestion
The search support synonyms—for example, try searching for "hamburger" or "logout."
```
